src/internal/operations/test/exchange_test.go

```diff
@@ -971,6 +975,9 @@ func (suite *ExchangeRestoreIntgSuite) TestRestore_Run_exchangeWithAdvancedOptio
 			userID,
 			cIDs[path.EmailCategory])
 		require.NoError(t, err, clues.ToCore(err))
+		countItemsInRestore += len(collKeys[path.EmailCategory])
+
+		checkRestoreCounts(t, ctr, 0, 0, countItemsInRestore)
```

Comment:
I can't comment on it directly, but it seems like we should be making a new counter for every restore we do right? Otherwise the counter will have the cumulative count from all restores and that seems like it would be a bit difficult for readers to reason through if just skimming the code

Reply:
The counters are unique to each run.  I could rename them if that'll ease confusion.

Reply:
no prob!